public/sitemap.xml

```diff
@@ -339,6 +339,36 @@
     >daily</changefreq><priority>0.6</priority></url><url><loc
     >https://www.grouparoo.com/integrations/destinations/zendesk</loc><changefreq
     >daily</changefreq><priority>0.6</priority></url><url><loc
+    >https://www.grouparoo.com/blog/author/evan_tahler</loc><changefreq
```

Comment:
In this PR, we decided that these author and tag pages were duplicative and left them out of the sitemap. https://github.com/grouparoo/www.grouparoo.com/pull/330

maybe they are in the whitelist or otherwise ignore in the test.

Reply:
Okay. That's fine. I kept the changes I made in terms of moving around the source of where these routes are generated, so it'd be trivial to add them to the sitemap in the future, should they become more unique.